src/include/cart/api.h

```diff
@@ -458,14 +458,16 @@ crt_hlc_get(void);
  *
  * \param[in] msg              remote HLC timestamp
  * \param[out] hlc_out         HLC timestamp
+ * \param[out] clock_drift     Optional returned clock drift. Only
+ *                             populated if -DER_HLC_SYNC is returned
```

Comment:
Would you mind naming this "offset"? I would like to keep the terminology consistent with the definition above crt_hlc_epsilon_set (see line 483 below).

The semantics could be (1) optional (i.e., may be NULL) and (2) if not NULL, always populated with the observed offset. See my comment in crt_hlc.c.

Reply:
Sure, will change